doc/Core.xml

```diff
@@ -6742,7 +6742,10 @@ tns1:RuleEngine/FieldDetector//.|tns1:RuleEngine/LineDetector//.
     </section>
     <section xml:id="_Ref212958632">
       <title>Get event properties</title>
-      <para>The [WS-BaseNotification] specification defines a set of optional WS-ResouceProperties. This specification does not require the implementation of the WS-ResourceProperty interface. Instead, the subsequent direct interface shall be implemented by an ONVIF compliant device in order to provide information about the FilterDialects, Schema files and topics supported by the device.</para>
+      <para>An ONVIF compliant device shall imlement this method to report supported event topics
+        and provide information about the FilterDialects and schema files.</para>
+      <para>Note, that configuration dependent events may not always be reported via this interface.
+        See e.g. section on GetSupportedRules for analytics event definitions. </para>
```

Comment:
e.g. Refer to GetSupportedRules specification regarding analytics event definitions.